dashboards-reports/package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "reports_dashboards",
-  "version": "1.0.0.0-rc1",
+  "name": "reports-dashboards",
```

Comment:
does changing the name here affect anything? i couldn't remember

Reply:
No, it shouldn't. I just changed it for format consistency. We no longer uses underscore for naming anywhere I think